src/components/Editor/Invitees/InviteesList.vue

```diff
@@ -221,3 +221,9 @@ export default {
 	},
 }
 </script>
+
+<style lang="scss" scoped>
+.invitees-list-button-group__button {
```

Comment:
the class does not exist yet, does it? you have to assign it to the button elements above

Reply:
Ouch ... I have no idea how to add the mentioned class to this ; can you help me a bit ? This would usefull for me for future contributions.
![2023-05-22_11-37](https://github.com/nextcloud/calendar/assets/33763786/67f9761e-1480-4e2c-b04b-71fe362cef6f)

Reply:
OK, I thought it was more complex or different, due to "vue" environment. So I did it. Can you test it ?